flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -89,6 +89,61 @@ under the License.
 		</dependency>
 	</dependencies>
 
+	<profiles>
+		<profile>
+			<!-- A profile that does everyting correctly:
```

Comment:
How about writing `Profile for packaging correct JAR files`. That let's people know it is actually for JAR files ;-)